--- TablePdoQueryConceptDao_orig.java	2015-10-21 20:41:09.000000000 +0200
+++ TablePdoQueryConceptDao.java	2015-10-21 22:41:15.492601878 +0200
@@ -103,8 +103,8 @@
 				insertSql = " insert into "
 						+ factTempTable;
 				
-				if (serverType.equalsIgnoreCase(DAOFactoryHelper.POSTGRESQL))
-					insertSql = " CAST(coalesce(char_param1, '0') as integer) ";
+				//if (serverType.equalsIgnoreCase(DAOFactoryHelper.POSTGRESQL))
+				//	insertSql = " CAST(coalesce(char_param1, '0') as integer) ";
 				//else 
 				//	insertSql += " (char_param1) ";
 				insertSql += "(char_param1) select distinct obs_concept_cd from ( "
